integration_tests/e2e/induction/updateCheckYourAnswersChangeLinks.cy.ts

```diff
@@ -126,7 +126,7 @@ context(`Change links on the Check Your Answers page when updating an Induction`
       .hasNoEducationalQualificationsDisplayed()
       .clickWantsToAddQualificationsChangeLink()
       .hasBackLinkTo(`/prisoners/${prisonNumber}/induction/check-your-answers`)
-      .selectHopingWorkOnRelease(YesNoValue.YES)
+      .selectWantToAddQualifications(YesNoValue.YES)
```

Comment:
This method name was changed (next file down) because it was wrong! (a shoddy copy and paste job by me! tsk tsk! 😬 )